main.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'classroom'
+require_relative 'options'
```

Comment:
- [ ] Just like the previous reviewer, I am unable to start up your app because here you are importing a file that does not exist in your app. Kindly fix your app and ensure it works correctly.

  ![vic-options](https://user-images.githubusercontent.com/36542986/157726419-02705dce-7f17-4a91-b261-2185ff993c96.jpg)